### ISSUE TITLE
Github Integration

### ISSUE THREAD

**Issue by amichuda** [nextcloud/deck#2760]
Is there some way to integrate Deck with a GitHub repository? So that any issues pop up automatically as a card on the board.

This would be an implementation similar to Git kraken boards:  https://www.gitkraken.com/boards

**Comment by putt1ck** [comment]
Sounds like something to implement with the API

**Comment by stefan-niedermann** [comment]
I agree with @putt1ck but i don't see this to be implemented at the Deck core.

There is already an existing [GitHub integration app](https://github.com/nextcloud/integration_github) which

> adds a dashboard item to see your most important GitHub notifications. It also adds a search provider for repositories, issues and pull requests.

I think this would be an ideal starting point for a deeper integration: The GitHub integration app could check whether Deck is installed & enabled and synchronize it. A first step could be a one-way-sync by displaying GitHub projects as read-only-boards in Deck, a second step could be a two-way-sync. :shrug:

**Comment by juliushaertl** [comment]
Yes, might be interesting if some integration could use the api access that is already setup with the github integration app. I could imagine that we'd either allow the user to paste a github issue link when creating a card and then just fetch the additional information. For automation this would be something for #1722

**Comment by AlphaCraft9658** [comment]
I bump this feature request!
